kml/pykmlib/bindings_test.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import datetime
+import pykmlib
+
+class PyKmlibAdsTest(unittest.TestCase):
+
+    def test_smoke(self):
+        category = pykmlib.CategoryData()
+        category.name.setDefault('Test category')
+        category.name['ru'] = 'Тестовая категория'
+        category.description.setDefault('Test description')
+        category.description['ru'] = 'Тестовое описание'
+        category.annotation.setDefault('Test annotation')
+        category.annotation['en'] = 'Test annotation'
+        category.imageUrl = 'https://localhost/123.png'
+        category.visible = True
+        category.authorName = 'Maps.Me'
+        category.authorId = '12345'
+        category.rating = 8.9
+        category.reviewsNumber = 567
+        category.lastModifiedTimestamp = int(datetime.datetime.now().timestamp())
```

Comment:
I'd call it either "last_modified" or "timestamp" (similar to the corresponding fields in Bookmarks and Tracks)